spec/reservation_manager_spec.rb

```diff
@@ -0,0 +1,199 @@
+require_relative "spec_helper"
+
+require "awesome_print"
+require "date"
+
+describe "manager" do
+  let (:manager) {
+    Hotel::ReservationManager.new
+  }
+  let (:block1) {
+    manager.make_block(5, "march 19, 2019", "march 23, 2019", 1)
+  }
+  let (:reservation1) {
+    manager.make_reservation("march 15, 2019", "march 20, 2019")
+  }
+  let (:reservation2) {
+    manager.make_reservation("march 17, 2019", "march 22, 2019")
+  }
+  let (:block2) {
+    manager.make_block(4, "march 21, 2019", "march 25, 2019", 2)
+  }
+
+  describe "Reservation_manager instantiation" do
+    it "is an instance of Reservation Manager" do
+      manager
+      expect(manager).must_be_kind_of Hotel::ReservationManager
+    end
+  end
+
+  describe "all_rooms method" do
+    it "can list all rooms in the hotel" do
+      manager
+      expect(manager.all_rooms.length).must_equal 20
+    end
+  end
+
+  describe "make_reservation method" do
+    it "can make a new reservation" do
+      reservation1
+      expect(reservation1.length).must_equal 1
+    end
+
+    it "raises an ArgumentError when there are no available rooms" do
+      expect {
+        (Hotel::ReservationManager::MAX_ROOMS + 1).times do
+          manager.make_reservation("march 15, 2019", "march 20, 2019")
+        end
+      }.must_raise ArgumentError
+    end
+  end
+
+  describe "reservations_by_date method" do
+    it "can list all the reservations for a specified date range" do
+      reservation1
+      reservation2
```

Comment:
Because the value of `reservation1` and `reservation2` is actually to call a method, you _do_ have to leave in the two above lines for the tests to pass. Maybe for this to make sense, instead of making it a `let` variable`, you might make this a helper method in the test (did you know you could have helper methods in tests?), or even just explicitly write it out in this test